state/src/storage/tx/matching_pools.rs

```diff
@@ -0,0 +1,206 @@
+//! Helpers for accessing information about matching pools in the database
+
+use common::types::wallet::OrderIdentifier;
+use libmdbx::{TransactionKind, RW};
+
+use crate::{storage::error::StorageError, POOL_TABLE};
+
+use super::StateTxn;
+
+// -------------
+// | Constants |
+// -------------
+
+/// The name of the global matching pool
+pub const GLOBAL_MATCHING_POOL: &str = "global";
```

Comment:
do we need to define this at the storage layer, it seems only used in tests. if we intend to use this at higher layers, it probably makes sense to define it there.

Reply:
yes ack, as I massage this into the relayer I'll find a better home for it (perhaps the `interface` layer)